pino-eventhub.js

```diff
@@ -10,6 +10,8 @@ const https = require('https')
 const debug = require('debug')('pino-eventhub')
 const Parse = require('fast-json-parse')
 
+
+
```

Comment:
these spaces are not needed. Are we running standard here? It should fail. If not, let's add it.